src/Db/Dump/PKSimDB_dump.txt

```diff
@@ -82970,7 +82970,7 @@ INSERT INTO tab_rate_generic_molecules VALUES('LinksCommon','MetabolizationSpeci
 INSERT INTO tab_rate_generic_molecules VALUES('LinksCommon','MetabolizationSpecific_Hill',372,'DRUG','C',0);
 INSERT INTO tab_rate_generic_molecules VALUES('LinksCommon','MetabolizationSpecific_MM',370,'DRUG','CP',0);
 INSERT INTO tab_rate_generic_molecules VALUES('LinksCommon','MetabolizationSpecific_MM',372,'DRUG','C',0);
-INSERT INTO tab_rate_generic_molecules VALUES('LinksCommon','MucosaActiveInfluxFromLumen_Hill',368,'DRUG','TM',1);
+INSERT INTO tab_rate_generic_molecules VALUES('LinksCommon','MucosaActiveInfluxFromLumen_Hill',366,'DRUG','TM',1);

```

Comment:
This was Target|Transporter. Now Source|Transporter

Reply:
no, that's not right. Is probably a relic from V9 (where all GI processes were defined with source in Mucosa and Influx-transports had the negative rate to compensate this)
Must be defined Lumen=>Mucosa (same as e.g. MucosaActiveInfluxFromLumen_MM)